.github/workflows/builder.yml

```diff
@@ -350,14 +353,23 @@ jobs:
       matrix:
         registry: ["ghcr.io/home-assistant", "docker.io/homeassistant"]
     steps:
-      - name: Checkout the repository
-        uses: actions/checkout@1af3b93b6815bc44a9784bd300feb67ff0d1eeb3 # v6.0.0
-
       - name: Install Cosign
         uses: sigstore/cosign-installer@faadad0cce49287aee09b3a48701e75088a2c6ad # v4.0.0
         with:
           cosign-release: "v2.2.3"
```

Comment:
Hm, we use 2.5.3 everywhere else, seems we missed bumping Core 🙈 

Shall I do a separate PR?